ovs/lib/migration.py

```diff
@@ -390,4 +390,20 @@ def _update_manifest_cache_size(_proxy_config_key):
             except Exception:
                 MigrationController._logger.exception('Integration of stats monkey failed')
 
+
+        ######################################
+        # Introduction of version key
+        if Configuration.get('key', default=False) not in ['community', 'enterprise']:
```

Comment:
Should be Configuration.get('/ovs/framework/edition', default=False) here. The entry 'key' will never exist (as it is never set)